Userland/Libraries/LibWeb/CSS/StyleProperties.cpp

```diff
@@ -5,6 +5,7 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include "LibWeb/CSS/StyleValue.h"
```

Comment:
This isn't needed - StyleValue.h is already included by StyleProperties.h

Reply:
These includes are automatically added by clangd for some reason. I will remove them anyway :^)

Reply:
No worries, I have the same issue with CLion sometimes.